extensions/amp-soundcloud/1.0/amp-soundcloud.css

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Since this css file is empty, let's delete it and remove the `hasCss` property in `bundles.config.extensions.json`